src/util/draftutil.ts

```diff
@@ -82,15 +83,20 @@ export const flattenSteps = (steps: Step[], pack: number): FlattenedStep[] => {
   return res;
 };
 
-export const defaultStepsForLength = (length: number): Step[] =>
-  new Array(length)
+export const defaultStepsForLength = (length: number): Step[] => {
+  const steps: DraftStep[] = new Array(length)
     .fill([
       { action: 'pick', amount: 1 },
       { action: 'pass', amount: 1 },
     ])
-    .flat()
-    .slice(0, length * 2 - 1) // Remove the final pass.
-    .map((action) => ({ ...action }));
+    .flat();
+
+  return normalizeDraftSteps(steps).map((step) => ({
+    action: step.action,
+    //We know not null here
+    amount: step.amount!,
```

Comment:
For whatever reason Step type doesn't allow amount to be null but DraftStep does. I kinda think the return type should be DraftStep anyways, since this isn't doing anything with pick/cardsInPack properties which are part of Step.